### ISSUE TITLE
[S3] [Minio] Error when creating new application: "Object not found key(xxxx)"

### ISSUE THREAD

**Issue by SimonXming** [spinnaker/spinnaker#4513]
### Issue Summary:
We are using Minio with NFS as S3 storage, while creating a new application, UI raises `Couldn not create application: Object not found key(xxxx)`. 

But it actually got created. After a few minutes, the newly created application will appear in the application list page. Also, I checked the `Modify time` of `front50/applications/xxxx/application-metadata.json`. The file `Modify time` is greater than the time spinnaker called `getObject` (https://github.com/spinnaker/front50/blob/master/front50-s3/src/main/java/com/netflix/spinnaker/front50/model/S3StorageService.java#L119).

So I guess it means when spinnaker called `getObject`, file creation still in progress(Maybe because of the underlay NFS slowness). 

### Cloud Provider(s):
Kubernetes

### Environment:
I am running Spinnaker using the `gcr.io/spinnaker-marketplace` images to deploy into self-host Kubernetes.

### Feature Area (if this issue is UI/UX related, please tag `@spinnaker/ui-ux-team`):
Front50

### Additional Details:

```
2019-06-11 02:59:55.071 WARN 1 --- [0.0-8080-exec-3] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [com.netflix.spinnaker.front50.exception.NotFoundException: Object not found (key: mingv2)]
2019-06-11 02:59:55.919 WARN 1 --- [ool-4-thread-13] .s.f.m.a.DefaultApplicationPermissionDAO : Unexpected stale read for mingv2 (current: Tue Jun 11 02:59:55 GMT 2019, expected: Tue Jun 11 02:59:55 GMT 2019)
```

**Comment by error418** [comment]
Which version of Spinnaker are you running? For more information about this issue see #4457

**Comment by SimonXming** [comment]
> Which version of Spinnaker are you running? For more information about this issue see #4457

I thinks it's `1.14.2`

**Comment by error418** [comment]
Try upgrading. The issue is fixed since 1.14.3

**Comment by SimonXming** [comment]
> Try upgrading. The issue is fixed since 1.14.3

Thanks, we will try that. 😄

**Comment by SimonXming** [comment]
@error418 `latest`(`1.14.5`) fix the issue. Thanks :)
